File: simple-rot13-dot-5-cypher/simple-rot13-dot-5-cypher.js
function ROT135(input) {
return input.split("").map(function(t){return changeChr(t)}).join("")
}

function changeChr(j){
if(j.match(/[a-z]/)){return String.fromCharCode((j.charCodeAt()+13-97)%26 + 97)}
else if(j.match(/[A-Z]/)){return String.fromCharCode((j.charCodeAt()+13-65)%26 + 65)}
else if (j.match(/[0-9]/)){return (parseInt(j)+5)%10}
else{return j}
}

File: simple-rot13-dot-5-cypher/simple-rot13-dot-5-cypher.py
from string import maketrans
def ROT135(input):
    return input.translate(maketrans("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789", "NOPQRSTUVWXYZABCDEFGHIJKLMnopqrstuvwxyzabcdefghijklm5678901234"))    